requirements-travis.txt

```diff
@@ -1,5 +1,6 @@
 # All pip installable requirements pinned for Travis CI
-fabric==1.11.1
+fabric==1.11.1; python_version <= '2.7'
+Fabric3==1.13.1.post1; python_version >= '3.4'
```

Comment:
As much as I think the remote module needs some rewriting to not depend on fabric or paramiko, sure.